ManageWikiExtensions.php

```diff
@@ -593,6 +593,14 @@
 		'requires' => [],
 		'section' => 'parserhooks',
 	],
+	'linkcards' => [
+		'name' => 'LinkCards',
+		'linkPage' => 'https://www.mediawiki.org/wiki/Extension:LinkCards',
+		'description' => 'Adds two parser functions for the easy display of grids of clickable image-and-title cards.',
```

Comment:
Remove; we don't hardcode descriptions. It will automatically use descriptionmsg from extension.
```suggestion
```